config/alias.php

```diff
@@ -44,6 +44,11 @@ function ppp($var)
     return (Tools::p($var));
 }
 
+function dump($var)
+{
+    return \Symfony\Component\VarDumper\VarDumper::dump($var);
+}
+
 function ddd($var)
 {
     Tools::d($var);
```

Comment:
Wasn't the purpose of this PR to provide another dump function rather than replace it?
And does Symfony's VarDumper `die()` as well?